scss/size.scss

```diff
@@ -4,6 +4,14 @@ $title-decoration-height: 90px;
 $title-decoration-width: 70px;
 $title-padding-left: 50px;
 $tech-logo-size: 100px;
+$tech-margin: 5px;
+%tech-container-size {
```

Comment:
I think that this isn't necessary because you can define in `.tech` class, so please delete it